src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java

```diff
@@ -2451,14 +2459,20 @@ public HyperwalletTransferMethod createTransferMethod(String jsonCacheToken, Str
      * Upload documents
      *
      * @param userToken userToken for which documents to be uploaded
-     * @param multiPart multipart FormdataMultipart to get uploaded
+     * @param uploadData HyperwalletVerificationDocument to get uploaded
      * @return HyperwalletUser user object with document upload status
      */
-    public HyperwalletUser documentUpload(String userToken, FormDataMultiPart multiPart) {
+    public HyperwalletUser uploadUserDocuments(String userToken, List<HyperwalletVerificationDocument> uploadData) {
+        Multipart multipart = new Multipart();
         if (StringUtils.isEmpty(userToken)) {
             throw new HyperwalletException("User token is not present");
         }
-        return apiClient.put(url + "/users/" + userToken, multiPart, HyperwalletUser.class);
+        try {
```

Comment:
Shouldn't validate the `uploadData` parameter like it was done in the stakeholder method?

Reply:
added validations for uploadData